.github/workflows/build-cinder-operator.yaml

```diff
@@ -0,0 +1,200 @@
+name: Cinder Operator image builder
+
+on:
+  push:
+    branches:
+      - '*'
+    paths-ignore:
+      - .gitignore
+      - .pull_request_pipeline
+      - changelog.txt
+      - kuttl-test.yaml
+      - LICENSE
+      - Makefile
+      - OWNERS
+      - PROJECT
+      - README.md
+      - .github/
+      - build/
+      - docs/
+      - tests/
+
+env:
+  imageregistry: 'quay.io'
+  imagenamespace: ${{ secrets.IMAGENAMESPACE || secrets.QUAY_USERNAME }}
+  latesttag: latest
+
+jobs:
+
+  check-secrets:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check secrets are set
+        id: have-secrets
+        if: "${{ env.imagenamespace != '' }}"
+        run: echo "::set-output name=ok::true"
+    outputs:
+      have-secrets: ${{ steps.have-secrets.outputs.ok }}
+
+  build-cinder-operator:
+    name: Build cinder-operator image using buildah
+    runs-on: ubuntu-latest
+    needs: [check-secrets]
+    if: needs.check-secrets.outputs.have-secrets == 'true'
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Get branch name
+      id: branch-name
+      uses: tj-actions/branch-names@v5
+
+    - name: Set latest tag for non master branch
+      if: "${{ steps.branch-name.outputs.current_branch != 'master' }}"
+      run: |
+        echo "latesttag=${{ steps.branch-name.outputs.current_branch }}-latest" >> $GITHUB_ENV
+
+    - name: Buildah Action
+      id: build-cinder-operator
+      uses: redhat-actions/buildah-build@v2
+      with:
+        image: cinder-operator
+        tags: ${{ env.latesttag }} ${{ github.sha }}
+        containerfiles: |
+          ./Dockerfile
+
+    - name: Push cinder-operator To ${{ env.imageregistry }}
+      uses: redhat-actions/push-to-registry@v2
+      with:
+        image: ${{ steps.build-cinder-operator.outputs.image }}
+        tags: ${{ steps.build-cinder-operator.outputs.tags }}
+        registry: ${{ env.imageregistry }}/${{ env.imagenamespace }}
+        username: ${{ secrets.QUAY_USERNAME }}
+        password: ${{ secrets.QUAY_PASSWORD }}
+
+  build-cinder-operator-bundle:
+    needs: [ check-secrets, build-cinder-operator ]
+    name: cinder-operator-bundle
+    runs-on: ubuntu-latest
+    if: needs.check-secrets.outputs.have-secrets == 'true'
+
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.17.x
```

Comment:
just to note , we used 1.18.x in other repos

Reply:
Ack